projectional/src/main/java/jetbrains/jetpad/projectional/cell/ProjectionalRoleSynchronizer.java

```diff
@@ -34,6 +34,7 @@
   <ContentT> void supportListContentKind(ContentKind<ContentT> kind, Function<ContentT, List<SourceT>> fromContent);
   void setOnLastItemDeleted(Runnable action);
   void setPlaceholderText(String text);
+  void disablePlaceholder(Runnable onLastItemDeleted);
```

Comment:
This is needed to force client specifying what to do when last item deleted. This emphasizes that the default action won't work (it yields NPE in fact).